packages/block-editor/src/components/link-control/index.js

```diff
@@ -408,8 +408,7 @@ function LinkControl( {
 						{ ! showActions && (
 							<div className="block-editor-link-control__search-enter">
 								<Button
-									// TODO: Switch to `true` (40px size) if possible
-									__next40pxDefaultSize={ false }
+									__next40pxDefaultSize
```

Comment:
**Before:**
![link-control-1-before](https://github.com/user-attachments/assets/9cee6acf-445e-4a44-ab1a-9f1d334f1fd2)

**After:**
![link-control-1-after](https://github.com/user-attachments/assets/8921bbd2-2dd5-4b06-90c8-fb65fb531f05)

Reply:
Doesn't look right. We'll need to fix the positioning to match up on the original.

Reply:
Let's also test creating a link in a standard paragraph block 🙏

Reply:
Test creating link in a standard paragraph block.

**Before:**
![link-control-1-before](https://github.com/user-attachments/assets/ae77226b-e34d-4d11-a997-cc6289680c56)

**After:**
![Uploading link-control-1-after.png…]()

Reply:
Turns out we'll be addressing this file in #65158, so you can ignore it here 👍

Reply:
@PARTHVATALIYA you should be able to merge latest `trunk` into this branch and ignore changes in this file